### PR TITLE
Feature/add complex method

### PR DESCRIPTION
**Pull request by JaimLus**
## Summary
- Complex method added

## Trello Card
https://trello.com/c/XRYc6xcD/25-agregar-m%C3%A9todos-m%C3%A1s-complejos-a-los-repositorios

### CODE REVIEW

**Review by Jimenarosello — DISMISSED** [review]
Nice job! 😊

File: src/main/java/wolox/training/repositories/UserRepository.java [hunk 1]
```diff
@@ -16,4 +18,15 @@
...
 
+    /**
+     * Method that obtain all users if your birthdate be between two date
+     *
+     * @param maxDate:        Max date ({@link LocalDate})
+     * @param minDate:        Min date ({@link LocalDate})
+     * @param nameContaining: Name containing of the user (String)
+     * @return {@link User} list
+     */
+    List<User> findByBirthdateLessThanEqualAndBirthdateGreaterThanEqualAndNameContainingIgnoreCase(
```
Comment (marcegl91) [inline src/main/java/wolox/training/repositories/UserRepository.java:29]: You can do this a little bit shorter with between:

https://docs.spring.io/spring-data/data-jpa/docs/current/reference/html/#jpa.query-methods.query-creation

But this is ok.

**Review by marcegl91 — DISMISSED** [review]
:tada:

**Review by marcegl91 — APPROVED** [review]

**Review by Jimenarosello — APPROVED** [review]
😄 🚀

**Pull request merged**